### PR TITLE
Adds a popup on crash, wait for it to close to quit the game

### PR DESCRIPTION
**Pull request by StraToN**

### CODE REVIEW

File: addons/escoria-core/ui_library/crash_popup/crash_popup.tscn [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[gd_scene load_steps=2 format=2]
```
Comment (dploeger) [inline addons/escoria-core/ui_library/crash_popup/crash_popup.tscn:1]: Why use a common design here? AcceptDialog already supports a dialog_text without any custom things.

File: addons/escoria-ui-9verbs/game.gd [hunk 2]
```diff
@@ -224,9 +224,31 @@ func pause_game():
...
+
+
+func show_crash_popup(files = []) -> void:
+	connect("crash_popup_confirmed", escoria, "quit", 
+		[], CONNECT_ONESHOT)
+	var crash_popup = escoria.resource_cache.get_resource(
+		"res://addons/escoria-core/ui_library/crash_popup/crash_popup.tscn"
+	).instance()
+	add_child(crash_popup)
+	move_child(crash_popup, 0)
```
Comment (dploeger) [inline addons/escoria-ui-9verbs/game.gd:243]: Why is this line required?

File: addons/escoria-ui-simplemouse/game.gd [hunk 3]
```diff
@@ -209,3 +209,24 @@ func _on_action_finished():
...
+
+
+func show_crash_popup(files = []) -> void:
+	connect("crash_popup_confirmed", escoria, "quit", 
+		[], CONNECT_ONESHOT)
+	var crash_popup = escoria.resource_cache.get_resource(
+		"res://addons/escoria-core/ui_library/crash_popup/crash_popup.tscn"
+		).instance()
+	add_child(crash_popup)
+	move_child(crash_popup, 0)
```
Comment (dploeger) [inline addons/escoria-ui-simplemouse/game.gd:221]: Why is this line required?

**Review by dploeger — CHANGES_REQUESTED** [review]
Awesome work! Thanks a lot. Just some smaller remarks.

File: addons/escoria-core/ui_library/crash_popup/crash_popup.tscn (hunk 1 above)
Comment (StraToN) [reply addons/escoria-core/ui_library/crash_popup/crash_popup.tscn:1]: Indeed! Thanks!

File: addons/escoria-ui-9verbs/game.gd (hunk 2 above)
Comment (StraToN) [reply addons/escoria-ui-9verbs/game.gd:243]: Mistake. The idea was originally to ensure the popup is drawn over everything, but this was not correct anyway. Thanks!

File: addons/escoria-ui-simplemouse/game.gd (hunk 3 above)
Comment (StraToN) [reply addons/escoria-ui-simplemouse/game.gd:221]: Mistake. The idea was originally to ensure the popup is drawn over everything, but this was not correct anyway.

**Review by dploeger — APPROVED** [review]
Great work! 🎁

**Comment by StraToN** [comment]
Thanks!
